modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -217,6 +223,16 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params& cfgContextConfig(cv::util::any&& ctx_cfg) {
```

Comment:
`cfgContextConfig` is something strange ("config" thing appeared twice), maybe `cfgContextParams` works better?

Reply:
I think it's more like `configure context configuration`. Renamed to `cfgContextParams`